jest.config.js

```diff
@@ -2,9 +2,8 @@ module.exports = {
   roots: ["<rootDir>"],
   clearMocks: true,
   testMatch: ["**/(__tests__|src)/**/?(*.)+(spec|test).+(ts|tsx|js)"],
-  transform: {
-    "^.+\\.(ts|tsx)?$": "ts-jest",
-  },
+  preset: "ts-jest/presets/default-esm",
+  extensionsToTreatAsEsm: ['.ts'],
```

Comment:
Isn't this default?

Reply:
The default is `ts-jest`, which outputs CJS modules, while `ts-jest/presets/default-esm` outputs ESM.